Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/SetPoints/ChilledWaterSupply.mo

```diff
@@ -0,0 +1,220 @@
+﻿within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.SetPoints;
+block ChilledWaterSupply
+  "Sequences to generate setpoints of chilled water supply temperaturea and the pump differential static pressure"
+
+  parameter Modelica.SIunits.PressureDifference dpChiWatPumMin(
+    final min=0,
+    displayUnit="Pa") = 34473.8
+    "Minimum chilled water pump differential static pressure, default 5 psi";
+  parameter Modelica.SIunits.PressureDifference dpChiWatPumMax(
+    final min=dpChiWatPumMin,
+    displayUnit="Pa")
+    "Maximum chilled water pump differential static pressure";
+  parameter Modelica.SIunits.ThermodynamicTemperature TChiWatSupMin(
+    displayUnit="K")
```

Comment:
@milicag I think it would be more consistent as the temperature parameter has to be assigned in Kelvin, like below I can give 288.706 but cannot assign it with 273.15+15.556.

Reply:
Fine by me.